src/xdocs/config_whitespace.xml

```diff
@@ -1437,7 +1444,32 @@ import static java.math.BigInteger.ZERO;
         <source>
 &lt;module name=&quot;NoWhitespaceAfter&quot;/&gt;
         </source>
+        <p>Example:</p>
+        <source>
+class Test {
 
+   public void lineBreak(String x) {
```

Comment:
Please remove extra space at the beginning of this line to align indentation

Reply:
done